js/src/redux/providers/registry/middleware.js

```diff
@@ -14,13 +14,40 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
+import { debounce } from 'lodash';
 import Contracts from '~/contracts';
 import subscribeToEvents from '~/util/subscribe-to-events';
 
 import registryABI from '~/contracts/abi/registry.json';
 
 import { setReverse, startCachingReverses } from './actions';
 
+const read = (chain) => {
+  const reverses = window.localStorage.getItem(`${chain}-registry-reverses`);
```

Comment:
It is not meant to be read by humans. :) 

I just want everything to align with our codebase de-facto standards we have in-place now. I don't think what we have is 100% perfect, but it also doesn't detract and is consistent at least in all the new code. A good step.

We already have a mix now of old that still needs to be migrated, I would not like to add another way of doing things and make things even worse. 

`logLevel` as an exception is tolerated since we can actually direct people to change manually in case of bugs and in this case since there is manual interaction it makes the simpler format better.